bin/plugin/commands/performance.js

```diff
@@ -31,6 +31,19 @@ const RESULTS_FILE_SUFFIX = '.performance-results.json';
  * @property {string=}  wpVersion   The WordPress version to be used as the base install for testing.
  */
 
+/**
+ * A logging helper for printing steps and their substeps.
+ *
+ * @param {number} indent Value to indent the log.
+ * @param {any}    msg    Message to log.
+ * @param {...any} args   Rest of the arguments to pass to console.log.
+ */
+function logAtIndent( indent, msg, ...args ) {
+	const prefix = indent === 0 ? '▶ ' : '> ';
```

Comment:
interesting change here. previously this was a different character - a non-breaking space no? both seem fine, but I'm curious what led to the change and to the initial choice.

Reply:
Hah, it was just me overthinking as usual. 😅 The nbsp felt right initially, so the title can't fully separate from the prefix char. I realized, though, that it's **highly** unlikely that it will ever happen, so I categorized it as code noise and removed it.